runtime/src/accounts_db.rs

```diff
@@ -7599,6 +7600,7 @@ impl AccountsDb {
                     None
                 },
                 zero_lamport_accounts: flavor.zero_lamport_accounts(),
+                dir_for_temp_cache_files: accounts_hash_cache_path,
```

Comment:
I guess this is ok. I was thinking of a peer directory to full/incremental hash cache files folders. These particular folders (full/incremental hash cache files) are scanned for all files and stragglers are deleted after each hash calc is run. Depending on the lifetimes of the hash cache objects, they could overlap with the creation & freeing of these temp files. Probably, this will work ok.